app/components/manage-intermissions/edit.js

```diff
@@ -79,4 +84,78 @@ export default class manageIntermissionsEditComponent extends Component {
     yield intermission.destroyRecord();
     this.args.onClose();
   }
+
+  //position stuff
+  get positionOptions() {
+    return [
+      { code: 'before', name: this.intl.t('manageIntermissions.beforeAp'), conceptUuid: "9c9be842-236f-4738-b642-f4064c86db51" },
+      { code: 'during', name: this.intl.t('manageIntermissions.duringAp'), conceptUuid: "4790eec5-acd2-4c1d-8e91-90bb2998f87c" },
+      { code: 'after', name: this.intl.t('manageIntermissions.afterAp'), conceptUuid: "267a09cc-5380-492d-93ad-697b9e99f032" }
```

Comment:
I'd recommend defining the uuids as constants so they can have a meaningful name

Reply:
I think actually just passing the concept scheme here instead is the better solution